corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html

```diff
@@ -3,7 +3,7 @@
 {% load compress %}
 {% load i18n %}
 
-{% requirejs_main_b5 'case_search/js/bootstrap5/profile_case_search' %}
+{% requirejs_main_b5 'case_search/js/bootstrap3/profile_case_search' %}
```

Comment:
```suggestion
{% requirejs_main_b5 'case_search/js/bootstrap5/profile_case_search' %}
```

Reply:
hmm, I should take a look at doing this rename automatically in the tool

Reply:
Thank you Biyeun! These two changes are flagged in the migration log. 
```
Flagged Code:

{% extends 'hqwebapp/bootstrap3/base_section.html' %}

Guidance:
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
This template extends a bootstrap 3 template.
```
So they should be changed in migrating view stage? (Though it doesn't matter if we change it now)

Reply:
It looks like that flag is slightly different: it's for the `extends`, not the `requirejs_main_b5`. But yeah, it doesn't matter if you make the change now or later.

Reply:
Thank you @orangejenny . I think Biyeun is correct. I go back to my old initial split PR and realized the reference to b5 js file will happen when the command line tool split js file. I will apply Biyeun's suggestion!